### PR TITLE
Implement exec command

### PR DESCRIPTION
**Pull request by Furisto**
Solves #20

### CODE REVIEW

File: src/tty.rs [hunk 1]
```diff
@@ -18,9 +19,10 @@ use crate::stdio::FileDescriptor;
 pub fn setup_console_socket(
     container_dir: &Path,
     console_socket_path: &Path,
+    socket_name: &str,
 ) -> Result<FileDescriptor> {
-    let csocket = "console-socket";
```
Comment (utam0k) [inline src/tty.rs:22]: :+1:

File: src/container/builder_impl.rs [hunk 2]
```diff
@@ -39,7 +39,9 @@ pub(super) struct ContainerBuilderImpl {
 impl ContainerBuilderImpl {
     pub(super) fn create(&mut self) -> Result<()> {
         if let Process::Parent(_) = self.run_container()? {
-            std::process::exit(0);
+            if self.init {
```
Comment (utam0k) [inline src/container/builder_impl.rs:42]: Can I ask you to add a comment to the struct explaining init?

Comment (Furisto) [reply src/container/builder_impl.rs:54]: @utam0k Added some more explanations. Let me know what you think.

File: src/container/builder_impl.rs [hunk 3]
```diff
@@ -21,25 +21,39 @@ use crate::{
 use super::{Container, ContainerStatus};
 
 pub(super) struct ContainerBuilderImpl {
+    /// Flag indicating if an init or a tenant container should be created
```
Comment (utam0k) [inline src/container/builder_impl.rs:24]: :+1:

**Review by utam0k — APPROVED** [review]
Superb!

**Pull request merged**